pxr/imaging/plugin/hdRpr/python/generateRenderSettingFiles.py

```diff
@@ -434,10 +434,14 @@ def houdini_parm_name(name):
                 }
             },
             {
-                'name': 'quality:enableReStir',
-                'ui_name': 'Enable ReStir',
-                'help': 'Enable ReStir (HybridPro only)',
-                'defaultValue': True,
+                'name': 'quality:reservoirSampling',
+                'ui_name': 'Reservoir Sampling (HybridPro only)',
+                'defaultValue': 'PathSpace',
+                'values': [
+                    SettingValue('Off'),
+                    SettingValue('ScreenSpace'),
+                    SettingValue('PathSpace'),
+                ],
```

Comment:
Sorry to be pedantic about help strings, but can we add an explanation of what this does with the "help" option?